ui/src/types/globalConfig/pages.ts

```diff
@@ -149,14 +149,27 @@ export const InputsPageTableSchema = z
     })
     .strict();
 
+export const InputsPageSchema = z
+    .union([InputsPageRegular, InputsPageTableSchema])
+    .optional()
+    .refine((data) => {
+        if (!data) {
+            return false;
+        }
+        if ('table' in data) {
+            return InputsPageTableSchema.safeParse(data).success;
+        }
+        return InputsPageRegular.safeParse(data).success;
```

Comment:
it is already parsed. There is no way success can be false. `.refine` does not narrow type of union